src/org/apache/xerces/dom/NodeImpl.java

```diff
@@ -239,6 +242,9 @@ public Node appendChild(Node newChild) throws DOMException {
     	return insertBefore(newChild, null);
     }
 
+    public DOMLocator getLocator() { return locator; }
```

Comment:
not sure of project style; but I tend to avoid single line methods. That, is move return to a new line